### ISSUE TITLE
PowerUps Module - Expand available powerups and functionality to be better.

### ISSUE THREAD

**Issue by mibby** [Eredrim/pvparena#57]
I came across this Splegg minigame plugin that has some unique powerups.
https://www.spigotmc.org/resources/splegg-minigame.26600/

The current powers in the PowerUps module are a bit lacking, probably because it was restricted by what was possible 6+ years ago. Would it be possible to implement some more fun powerups to be used in games such as Run and Spleef?

**Comment by Eredrim** [comment]
Yes PowerUps is not the funniest mod of PVPArena, but several weeks ago it didn't work at all. Currently several modules keep to be in this case: they badly work or worse, they don't work at all. So, you will understand add more fun to a module is not my coming priorities. For information, here's the kind of roadmap I try to follow:

1. Fixing main plugin issues (done)
2. Add autocompletion to commands (done)
3. Improve documentation (in progress)
4. Releasing 1.15 version (planned for this month)
5. Fixing modules errors (later)
6. Add enhancements (later again)

**Comment by Eredrim** [comment]
PowerUps has been released in V2 with a fully rewritten documentation: https://github.com/Eredrim/pvparena/blob/master/doc/mods/powerups.md
